### DISCUSSION TITLE
Version Control in Sharepoint

### DISCUSSION THREAD

**Discussion by riversource** [abraunegg/onedrive#2030] [Q&A] [answered]
Hi,

I'm using the OneDrive client v2.4.19 on an Ubuntu 22.04 server installation in order to sync files into a SharePoint document library. Before I used the OneDrive client, I did the same thing using rclone.

In general, it works. But I have version control enabled in the Sharepoint Library, and syncing with this OneDrive client causes the removal of all previous versions of the files. Syncing with rclone adds a new version in case the local file was changed.

I'm pretty sure I missed a configuration item, but I currently have no clue. Any ideas?

**Comment by abraunegg** [comment, marked as answer]
@riversource 
> I'm pretty sure I missed a configuration item, but I currently have no clue. Any ideas?

This is most likely due to the issues with SharePoint and the OneDrive API. If you run the client in verbose mode (`--verbose`) you will be able to see more on what the client is doing to understand why you are seeing this type of activity.
